features/donation.feature

```diff
@@ -0,0 +1,198 @@
+      | NUMERO_CARTE | 1111 2222 3333 4444 |
+      | CVVX         | 123                 |
+    And I select "12" from "MOIS_VALIDITE"
+    And I select "18" from "AN_VALIDITE"
+    And I press "VALIDER"
+    Then I wait 5 second until I see "PAIEMENT ACCEPTé"
+    And I should be on "https://preprod-tpeweb.paybox.com/cgi/MYtraitetransaction.cgi"
+
+    When I click on the "1" "img" element
+    Then I should see "Votre soutien compte beaucoup pour moi."
+
+    Given I am on "/don"
+    And wait 1 second until I see "Continuer"
+    And I click the "donation-monthly_label" element
+    When I press "Continuer"
+    Then I should be on "/don/coordonnees?montant=50&abonnement=1"
+
+    When I fill in the following:
+      | app_donation_gender | male                     |
+      | Nom                 | Jean                     |
+      | Prénom              | Dupont                   |
+      | Adresse email       | jean.dupont@en-marche.fr |
+      | Code postal         | 75001                    |
+      | Ville               | Paris                    |
+      | Adresse postale     | 1 allée vivaldie         |
+      | Numéro de téléphone | 0123456789               |
+    And I click the "donation_check_label" element
+    And I click the "donation_check_nationality_label" element
+    And I press "Je donne"
+    Then I should be on "/don/coordonnees?montant=50&abonnement=1"
+    And I should see "Vous faites déjà un don mensuel à La République En Marche ! Vous pouvez vous rendre sur votre profil pour en modifier le montant ou faire un nouveau don unique."
+
+    When I follow "faire un nouveau don unique"
+    Then I should be on "/don/coordonnees?montant=50"
+    And the "app_donation_gender" field should contain "male"
+    And the "Nom" field should contain "Jean"
+    And the "Prénom" field should contain "Dupont"
+    And the "Adresse email" field should contain "jean.dupont@en-marche.fr"
+
+  @javascript
+  Scenario: The logged user can subscribe to donate each month successfully but can't have a second subscription without unsubscribe before
+    Given the following fixtures are loaded:
+      | LoadAdherentData |
+    And I am logged as "jacques.picard@en-marche.fr"
+    And I am on "/don"
+    And I press "OK"
+    And wait 1 second until I see "Continuer"
+    When I click the "donation-monthly_label" element
+    And I press "Continuer"
+    Then I should be on "/don/coordonnees?montant=50&abonnement=1"
+
+    When I click the "donation_check_label" element
+    And I click the "donation_check_nationality_label" element
+    And I press "Je donne"
+    Then I wait 5 second until I see "Numéro de carte"
+    And I should be on "https://preprod-tpeweb.paybox.com/cgi/MYpagepaiement.cgi"
+
+    When I fill in the following:
+      | NUMERO_CARTE | 1111 2222 3333 4444 |
+      | CVVX         | 123                 |
+    And I select "12" from "MOIS_VALIDITE"
+    And I select "18" from "AN_VALIDITE"
+    And I press "VALIDER"
```

Comment:
`Valider`

Reply:
Not possible , it's the name of the button because the label is an image